### PR TITLE
Add new contact UI

### LINKED ISSUE #5: Create a new contact UI

### PR DESCRIPTION
**Pull request by Thejus-Paul**
Fixes #5 
- Added create contact form and its constants.
- Added create contact component.

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Contacts/Pane/CreateContact.jsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import React from "react";
+
+import { Pane, Typography } from "neetoui/v2";
+import PropTypes from "prop-types";
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Contacts/Pane/CreateContact.jsx:4]: Same here.

File: app/javascript/src/components/Dashboard/Contacts/Pane/CreateContact.jsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+import Form from "./Form";
+
+const CreateContact = ({
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Contacts/Pane/CreateContact.jsx:8]: Let's not keep it inside a folder called `Pane`.

File: app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx [hunk 3]
```diff
@@ -0,0 +1,96 @@
+import React from "react";
+
+import { Check } from "@bigbinary/neeto-icons";
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx:3]: Please use aliases.

File: app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+  VALIDATION_SCHEMA,
+  INITIAL_ROLE_VALUES,
+} from "./constants";
+
+const Form = ({ onClose, handleSubmit, initialContactValues }) => {
+  const populateSelectOptions = options =>
+    options.map(tag => ({
+      label: tag,
+      value: tag,
+    }));
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx:20]: Can we name the function better?

File: app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+      {({ setFieldValue }) => (
+        <FormikForm className="w-full">
+          <Pane.Body className="space-y-6">
+            <div className="flex w-full space-x-4">
+              <Input
+                label="First Name"
+                name="firstName"
+                placeholder="Enter first name"
+                size="large"
+                required
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx:37]: Do we need `required` here? and also to the fields below?

File: app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx [hunk 6]
```diff
@@ -0,0 +1,96 @@
+import React from "react";
+
+import { Check } from "@bigbinary/neeto-icons";
+import { Formik, Form as FormikForm } from "formik";
+import { Button, Pane } from "neetoui/v2";
+import { Input, Select } from "neetoui/v2/formik";
+import PropTypes from "prop-types";
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx:7]: Same here.

**Review by karthiknmenon — COMMENTED** [review]
@Thejus-Paul Please check the suggestions.

File: app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx (hunk 5 above)
Comment (Thejus-Paul) [reply app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx:37]: In the Figma design, there were `*` at the end of every field so, I thought it was to put `required`. Should I just replace it with `*` in the label field and remove the `required`?

**Pull request merged**